DOCS/man/options.rst

```diff
@@ -4304,6 +4309,11 @@ Input
     for mouse key bindings and scripts which read mouse positions for platforms
     which do not support ``--native-touch=no`` (e.g. Wayland).
 
+``--input-dragging-deadzone=<N>``
```

Comment:
Should this be scaled by DPI?

Reply:
I don't think that would be useful.

Reply:
For edge dragging it is quite important to have sane size of the drag handle. IMHO it should be scaled.

Reply:
I know, it was comparison. Same as for edge dragging, amount of mouse travel before window enters dragging state could be adjusted by dpi, but if you guys don't agree, fine.

Reply:
Ok.